adapters/BUILD

```diff
@@ -0,0 +1,14 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+	name = "go_default_library",
```

Comment:
The docs at the [bazelbuild/go_rules](https://github.com/bazelbuild/rules_go) suggest that this is a special name required to make the go import statements work corretly.